projects/pokemon_game.py

```diff
@@ -28,41 +29,36 @@ def __init__(self, name, primary_type, max_hp, hp) -> None:
     def battle (self, computer_pokemon):
         # water > fire > grass > water
         if self.primary_type == computer_pokemon.primary_type:
-            print(f"Both Pokemon are type {self.primary_type}, it's a tie! They both loose 1 HP. \n")
-            user_pokemon.hp -= 1
-            computer_pokemon.hp -= 1
+            print(f"Both Pokemon are type {self.primary_type}, it's a tie! You both loose 5 HP. \n")
```

Comment:
**suggestion**: This is the first of many lines that are quite long. Up until now I've said keeping line length under 80 or 88 is optional but from now on I'm going to be enforcing it 😈. Let's keep it to under 80 going forward.

Python will automatically concatenate two strings that are next to each other, eg:

```py
(
    "Hello "
    "World"
)
```

Will give you `"Hello World"`

So you can use that to keep your longer strings under 80 characters:

```suggestion
            print(
                f"Both Pokemon are type {self.primary_type}, it's a tie!"
                f"You both loose 5 HP. \n"
            )
```

With the change I suggested before about using constants with your magic numbers, this would change to:

```suggestion
            print(
                f"Both Pokemon are type {self.primary_type}, it's a tie!"
                f"You both loose {HP_LOSS_DECREMENT} HP. \n"
            )
```

And, by the way, "loose" is the adjective meaning not firmly or tightly fixed in place, i.e. "a loose tooth". You want the verb "to lose":

```suggestion
            print(
                f"Both Pokemon are type {self.primary_type}, it's a tie!"
                f"You both lose {HP_LOSS_DECREMENT} HP. \n"
            )
```